intelmq/bots/BOTS

```diff
@@ -1,56 +1,56 @@
 {
     "Collector": {
         "Abuse.ch Feodo Tracker Domains": {
-            "description": "Abuse.ch Feodo Tracker Domains is the bot responsible to get the report from source of information.",
```

Comment:
The description has a pattern: starts always with the name of the bot, e.g. 'Abuse.ch Feodo Tracker Domains'